src/PlaneWaveBasis.jl

```diff
@@ -229,10 +232,12 @@ treated explicitly. In this case all guess densities and potential
 functions must agree with the crystal symmetries or the result is
 undefined.
 """
-function PlaneWaveBasis(model::Model, Ecut::Number;
+function PlaneWaveBasis(model::Model,
```

Comment:
I'm not sure this is the best place to add units. This is kind of the low-level constructor, which is only supposed to be used internally by DFTK. I think the better place would be the high-level constructor at

https://github.com/JuliaMolSim/DFTK.jl/blob/master/src/PlaneWaveBasis.jl#L232

In fact at that level I think it would be fair to just drop the type annotation `::Number` and pass whatever you get straight to `austrip` before passing it on to this constructor.

Reply:
This is the same constructor you're linking to.

Reply:
Indeed you are right. My bad.

With respect to the type annotation of `Ecut`, maybe a bit more of my reasoning: In general in Julia one wants to avoid "strange type unions" (i.e. ones which involve to some extent unrelated types) and `Union{Number, Quantity}` is to me one of these cases. The general argument is that if you start needing these `Union`s this points to your interface being more generic anyway and therefore typically there is no need to annotate the type. So that's why my second remark was to suggest dropping the type annotation for `Ecut`. ;)